CRM/Utils/System/Base.php

```diff
@@ -68,6 +68,34 @@ abstract public function isLoaded(): bool;
 
   abstract public function loadBootStrap($params = [], $loadUser = TRUE, $throwError = TRUE, $realPath = NULL);
 
+  /**
+   * Returns the Smarty template path to the main template that renders the content.
+   *
+   * In CMS contexts, this goes inside their theme, but Standalone needs to render the full HTML page.
+   *
+   * @var int|string $print
+   *   Should match a CRM_Core_Smarty::PRINT_* constant,
+   *   or equal 0 if not in print mode.
+   */
+  public static function getContentTemplate($print = 0): string {
+    switch ($print) {
```

Comment:
I think 'CRM/common/snippet.tpl' has been dropped? 

I have no personal attachment to `CRM/common/snippet.tpl` (or any of these print templates for that matter...)